python/semantic_kernel/connectors/ai/open_ai/semantic_functions/open_ai_chat_prompt_template.py

```diff
@@ -13,9 +13,11 @@
 
 
 class OpenAIChatPromptTemplate(ChatPromptTemplate[OpenAIChatMessage]):
-    def add_function_response_message(self, name: str, content: Any) -> None:
+    def add_function_response_message(self, name: str, content: Any, tool_call_id: Optional[str]) -> None:
```

Comment:
Let's give toolcallid a default None for backward compatibility

Reply:
I think we need to eventually move to having the `OpenAIChatMessage` support a List[ToolCall] because the model may return multiple tool calls. Right now we're only supporting the first, but if we get more than one, and don't send back all tool_call_ids, it'll fail.